torch_xla/csrc/aten_cuda_functions.cpp

```diff
@@ -11,6 +11,8 @@ static void fail(const char* name) {
 
 namespace c10::cuda {
 
+DeviceIndex device_count() noexcept { return 0; }
```

Comment:
IIRC, we unit test this function in python layer - do we need a unit test for the cpp layer?